### PR TITLE
Issue 568 angular integration

### PR DESCRIPTION
**Pull request by maheee**
I've updated the Angular integration in this way:

- updated the SettingsModel to reflect the current state described in README.md
- removed the service since it doesn't make too much sense as singleton
- moved the functions from the service to the component

Still, I think maybe the angular integration is best used as a working example to be copied and modified in the project where it is used. Right now I'm not sure how to expose all the events and methods in a reasonable way.

### CODE REVIEW

File: dist/angular-tagify.component.ts [hunk 1]
```diff
@@ -1,24 +1,49 @@
...
-  autocomplete?: boolean;
+  autoComplete?: {
+      enabled?: boolean;
+      rightKey?: boolean;
+  }
   whitelist?: string[];
   blacklist?: string[];
   addTagOnBlur?: boolean;
-  callbacks?: Object;
+  callbacks?: any;
```
Comment (yairEO) [inline dist/angular-tagify.component.ts:20]: why `any`? it can only be an `object`

File: dist/angular-tagify.component.ts [hunk 2]
```diff
@@ -1,24 +1,49 @@
...
-  'dropdown.enabled'?: number;
-  'dropdown.maxItems'?: string;
-  'dropdown.classname'?: string;
-  'dropdown.itemTemplate'?; Function;
+  keepInvalidTags?: boolean;
+  skipInvalid?: boolean;
+  backspace?: any;
+  originalInputValueFormat?: Function;
+  dropdown: {
+      enabled?: number;
```
Comment (yairEO) [inline dist/angular-tagify.component.ts:34]: `enabled` may be a number or `false`

File: dist/angular-tagify.component.ts [hunk 3]
```diff
@@ -1,24 +1,49 @@
...
+      rightKey?: boolean;
+  }
   whitelist?: string[];
   blacklist?: string[];
   addTagOnBlur?: boolean;
-  callbacks?: Object;
+  callbacks?: any;
   maxTags?: number;
+  editTags?: number;
+  templates?: {
```
Comment (yairEO) [inline dist/angular-tagify.component.ts:23]: there is 1 more template: `dropdownItemNoMatch `

File: dist/angular-tagify.component.ts [hunk 4]
```diff
@@ -1,24 +1,49 @@
...
+  mixTagsInterpolator?: string[];
+  mixTagsAllowedAfter?: RegExp;
   duplicates?: boolean;
   enforceWhitelist?: boolean;
-  autocomplete?: boolean;
+  autoComplete?: {
+      enabled?: boolean;
+      rightKey?: boolean;
+  }
   whitelist?: string[];
```
Comment (yairEO) [inline dist/angular-tagify.component.ts:17]: `whitelist` can be an Array of *Objects*, Or an Array of *Strings*

**Comment by yairEO** [comment]
If you know React, you might learn how I've exposed things in the wrapper I've written:
https://github.com/yairEO/tagify/blob/master/src/react.tagify.js

I just don't know Angular, **at all**...

**Comment by maheee** [comment]
Thanks for your help, I'll have a look at the react implementation, even though I don't know much react 😄

**Pull request merged**